src/Keboola/Console/Command/MassProjectQueueMigration.php

```diff
@@ -87,7 +91,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 self::COMPONENT_QUEUE_MIGRATION_TOOL,
                 '',
                 [
-                    'parameters' => []
+                    'parameters' => [
+                        'kill_processing_jobs' => true,
```

Comment:
no já nevím, to by mělo být taky na parametr ne? Na CSAS to bylo domluvený, že to zabijem, ale jinde to tak být nemusí.